t/11-image-ppm.t

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl
+# Do not add to makefile.am
+
+use strict;
+use warnings;
+use Test::More;
+
+use Try::Tiny;
+use File::Basename;
+use File::Path qw(make_path remove_tree);
+use Cwd;
+
+
+BEGIN {
+    unshift @INC, '..';
+}
+
+use OpenQA::Benchmark::Stopwatch;
+
+
+# optional but very useful
+eval 'use Test::More::Color';                 ## no critic
+eval 'use Test::More::Color "foreground"';    ## no critic
```

Comment:
sounds nice but I can't see a difference when trying it out locally on another test.